src/elements/hx-switch/index.js

```diff
@@ -67,6 +68,20 @@ export class HXSwitchElement extends HXFormControlElement {
         this.setAttribute('onlabel', newVal);
     }
 
+    /**
+     * Property reflecting the `toggled` state of the Switch component.
+     */
+    get toggled () {
+        return this.hasAttribute('toggled');
+    }
+    set toggled (isToggled) {
+        if (isToggled) {
+            this.setAttribute('toggled', '');
+        } else {
+            this.removeAttribute('toggled');
+        }
+    }
```

Comment:
Added a `toggled` attribute to provide consumers an option to check Switch Component state.